src/async-wrap.cc

```diff
@@ -508,6 +561,11 @@ void AsyncWrap::Initialize(Local<Object> target,
   env->SetMethod(target, "enablePromiseHook", EnablePromiseHook);
   env->SetMethod(target, "disablePromiseHook", DisablePromiseHook);
 
+  env->SetMethod(target, "asyncTaskScheduled", AsyncTaskScheduled);
+  env->SetMethod(target, "asyncTaskCanceled", AsyncTaskCanceled);
+  env->SetMethod(target, "asyncTaskStarted", AsyncTaskStarted);
+  env->SetMethod(target, "asyncTaskFinished", AsyncTaskFinished);
```

Comment:
Same here, I think it would be better to keep this code in the inspector cc files?